03-Bash/fix_owner/fix_owner.sh

```diff
@@ -0,0 +1,3 @@
+! #bin/bash
+
+echo "test fix_owner.sh"
```

Comment:
What is the purpose of this file?

Reply:
I created an empty file for the test. What should I do with this commit? Should I squash it?

Reply:
For these tasks the requirement was 1 task = 1 commit. Force pushes to your private branches are allowed.